src/OSPSuite.UI/Views/Charts/ChartEditorView.cs

```diff
@@ -57,18 +60,51 @@ public ChartEditorView(IMenuBarItemToBarItemMapper barItemMapper, IImageListRetr
 
          _barEditItemForUsedIn.SuperTip = new SuperToolTip().WithText(ToolTips.UseSelectedCurvesToolTip);
 
+         _barEditItemLinkSimulationObserved = new BarEditItem(_barManager)
+         {
+            Edit = repositoryItemCheckEditLinkSimulationAndObserved,
+            Alignment = BarItemLinkAlignment.Right,
+            AutoFillWidth = false,
+            Width = 20,
+            Caption = Captions.LinkDataToSimulations,
+            CaptionAlignment = HorzAlignment.Near,
+            PaintStyle = BarItemPaintStyle.Caption,
+            Hint = "test hint"
```

Comment:
Remove this and use a constant

Reply:
this is actually unecessary, was just a test. removed.